tox.ini

```diff
@@ -25,6 +25,7 @@ deps =
     pytest
     coverage
     semsimian
+    gilda
```

Comment:
this seems an unrelated change?

Reply:
tox -e py fails locally if this is absent.